lib/api/src/externals/function.rs

```diff
@@ -291,6 +317,7 @@ impl Function {
                 address,
                 kind: VMFunctionKind::Static,
                 vmctx,
+                function_ptr,
```

Comment:
We should move this out of this struct.

Reply:
This is the same as https://github.com/wasmerio/wasmer/pull/1739#discussion_r526446162 ; it may be possible but I'm not exactly sure how yet